language/en/common.php

```diff
@@ -94,7 +94,7 @@
 	'VIEW_ALL'				=> 'View All',
 	'VIEW_IDEA'				=> 'View Idea',
 	'VIEW_IDEA_AT'          => 'View idea at: %s', // Warning: submitted to db
-	'VIEW_VOTES'            => '%s points. Click to view votes',
+	'VIEW_VOTES'            => '%s point(s). Click to view votes',
```

Comment:
I already tried. This key is swapped back and forth by javascript when voting and couldn't get it to work out.